src/languages/en.js

```diff
@@ -98,6 +98,10 @@ export default {
     sidebarScreen: {
         newChat: 'New Chat',
         newGroup: 'New Group',
+        headerChat: 'Chats',
+        buttonSearch: 'Search',
+        buttonMySettings: 'My Settings',
+        fabNewChat: 'New Chat(Floating Action)',
```

Comment:
it wont be displayed on screen but would annouce via VoiceOver for disabled people. We should need to make it multi-language too. Because users can choose the VoiceOver language on their devices.

Reply:
Why would a blind user care that the new chat button is in a floating action or not? IMO it should just say `new chat`.